src/adapters/scene_direction.js

```diff
@@ -139,24 +139,36 @@ export default class SceneDirection {
   }
 
   showPolygon(route, vehicle) {
-    const geojson = {
-      "id": `route_${route.id}`,
-      "type": "line",
-      "source": `source_${route.id}`,
-      "layout": {
-        "line-join": "round",
-        "line-cap": "round",
-        "visibility": "visible"
-      },
-      "paint": {
-        "line-color": ["case",
-          ["boolean", ["feature-state", "isActive"], route.isActive],
-          MAIN_ROUTE_COLOR,
-          ALTERNATE_ROUTE_COLOR
-        ],
-        "line-width": 7
+    const geojson = (vehicle === "walking") ? {
+        "id": `route_${route.id}`,
+        "type": "symbol",
+        "source": `source_${route.id}`,
+        "layout": {
+          'icon-image': 'walking_bullet',
```

Comment:
here the same image is used whether the route is active or not. I think we would like to use grey dots on inactive routes, similarly to how a different "line-color" is used below for non-walking vehicle.

Reply:
I added a case expression here 👍 
And whenever a `toggle_route` is triggered, i implemented a `setLayoutProperty` to directly update the `icon-image`